src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -5232,6 +5235,42 @@ public static Value can_faxbot(ScriptRuntime controller, final Value arg) {
     return DataTypes.FALSE_VALUE;
   }
 
+  public static Value can_faxbot(ScriptRuntime controller, final Value arg1, final Value arg2) {
```

Comment:
This function shares most of its code with the one above: could you rewrite it so one calls the other (e.g. with null as the final value), or do the Value parsing first, and then have them both call `can_faxbot(MonsterData monster, String faxbot)`, one with a null faxbot?

Reply:
Sure. I did the latter.